index.html

```diff
@@ -76,50 +80,55 @@ <h2>Multi-Post
 				<li><a href="">javascript</a></li>
 			</ul>
 
-			<button class="btn">See Project</button>
+			<button type="button" class="btn">See Project</button>
+		</div>
```

Comment:
- [ ] [OPTIONAL] I congratulate you on applying the `hover` effect on the button 👏  👍 . But you should ensure that `UI` is professional. I suggest that you make the `button text` color `white` on hover. Read more to understand the [hover effect](https://www.w3schools.com/cssref/sel_hover.php).